app/src/forms/common/models/tables/user.js

```diff
@@ -40,11 +40,13 @@ class User extends Timestamps(Model) {
           query.where('idpCode', value);
         }
       },
-      filterUsername(query, value, exact = false) {
+      filterUsername(query, value, exact = false, caseSensitive = true) {
         if (value) {
-          if (exact) query.where('username', value);
+          if (exact && caseSensitive) query.where('username', value);
           // ilike is postgres case insensitive like
-          else query.where('username', 'ilike', `%${value}%`);
+          else if (exact && !caseSensitive) query.where('username', 'ilike', String(value).toLowerCase());
+          else if (!exact && caseSensitive) query.where('username', 'ilike', `%${value}%`);
+          else query.where('username', 'ilike', `%${String(value).toLowerCase()}%`);
```

Comment:
It looks like the line 46 comment is correct: PostgreSQL has a non-standard extension `ilike`, which is `like` but case-insensitive: https://www.postgresql.org/docs/13/functions-matching.html#FUNCTIONS-LIKE. I confirmed this with SQL queries against the DB, but haven't tried it in knex - hopefully it's the same!

So that would make lines 48 and 49 the same. So I think the problem there is that line 48 should be `like` instead of `ilike`, since it's case sensitive.

```suggestion
          else if (exact && !caseSensitive) query.where('username', 'ilike', value);
          else if (!exact && caseSensitive) query.where('username', 'like', `%${value}%`);
          else query.where('username', 'ilike', `%${value}%`);
```

Reply:
right, line 48 should be like, fixed it. And `.toLowerCase()` is not needed as well.